src/types/src/expectedPlayoutItems.ts

```diff
@@ -5,5 +5,7 @@ export type ExpectedPlayoutItemContent = ExpectedPlayoutItemContentVizMSE
 export interface ExpectedPlayoutItemContentBase {
 	/** Id of the rundown the items comes from */
 	rundownId: string
+	/** Id of the rundown playlist the items comes from */
+	playlistId: string
```

Comment:
This is not the appropriate place to define playlistId (or rundownId), as the blueprints will have to fill in something to get typescript to be happy. Playout gateway will then overwrite these values with real data.
This `ExpectedPlayoutItemContentBase` would be better to be kept in the non types side, or just within playout-gateway